src/dodal/devices/focusing_mirror.py

```diff
@@ -0,0 +1,145 @@
+                    LOGGER.warn(
+                        f"Failed to fetch {setpoint_v.name} Demand Accepted", exc_info=e
+                    )
+                    demand_accepted_status.set_exception(e)
+
+        setpoint_status.add_callback(setpoint_callback)
+        status = setpoint_status & demand_accepted_status
+        return status
+
+
+class VFMMirrorVoltages(Device):
+    _channel14_voltage_device: MirrorVoltageDevice = Component(
+        MirrorVoltageDevice, "BM:V14"
+    )
+    _channel15_voltage_device: MirrorVoltageDevice = Component(
+        MirrorVoltageDevice, "BM:V15"
+    )
+    _channel16_voltage_device: MirrorVoltageDevice = Component(
+        MirrorVoltageDevice, "BM:V16"
+    )
+    _channel17_voltage_device: MirrorVoltageDevice = Component(
+        MirrorVoltageDevice, "BM:V17"
+    )
+    _channel18_voltage_device: MirrorVoltageDevice = Component(
+        MirrorVoltageDevice, "BM:V18"
+    )
+    _channel19_voltage_device: MirrorVoltageDevice = Component(
+        MirrorVoltageDevice, "BM:V19"
+    )
+    _channel20_voltage_device: MirrorVoltageDevice = Component(
+        MirrorVoltageDevice, "BM:V20"
+    )
+    _channel21_voltage_device: MirrorVoltageDevice = Component(
+        MirrorVoltageDevice, "BM:V21"
+    )
+
+    voltage_lookup_table_path: str = (
+        "/dls_sw/i03/software/daq_configuration/json/mirrorFocus.json"
+    )
+
+    @property
+    def voltage_channels(self) -> list[MirrorVoltageDevice]:
+        return [
+            getattr(self, name)
+            for name in [
+                "_channel14_voltage_device",
+                "_channel15_voltage_device",
+                "_channel16_voltage_device",
+                "_channel17_voltage_device",
+                "_channel18_voltage_device",
+                "_channel19_voltage_device",
+                "_channel20_voltage_device",
+                "_channel21_voltage_device",
+            ]
+        ]
+
+
+class FocusingMirror(Device):
+    """Focusing Mirror"""
+
+    yaw_mrad: EpicsMotor = Component(EpicsMotor, "YAW")
+    pitch_mrad: EpicsMotor = Component(EpicsMotor, "PITCH")
+    fine_pitch_mm: EpicsMotor = Component(EpicsMotor, "FPMTR")
+    roll_mrad: EpicsMotor = Component(EpicsMotor, "ROLL")
+    vert_mm: EpicsMotor = Component(EpicsMotor, "VERT")
+    lat_mm: EpicsMotor = Component(EpicsMotor, "LAT")
+    jack1_mm: EpicsMotor = Component(EpicsMotor, "Y1")
+    jack2_mm: EpicsMotor = Component(EpicsMotor, "Y2")
+    jack3_mm: EpicsMotor = Component(EpicsMotor, "Y3")
+    translation1_mm: EpicsMotor = Component(EpicsMotor, "X1")
+    translation2_mm: EpicsMotor = Component(EpicsMotor, "X2")
+
+    stripe: EpicsSignal = Component(EpicsSignal, "STRP:DVAL", string=True)
+    # apply the current set stripe setting
+    apply_stripe: EpicsSignal = Component(EpicsSignal, "CHANGE.PROC")
+
+    bragg_to_lat_lookup_table_path: str = (
+        "CONFIGURE_ME"  # configured in per-mirror beamline-specific setup
+    )
```

Comment:
Should: If this is a required part of the device can you pass it in to the constructor?